docs/tutorial/first-app.md

```diff
@@ -0,0 +1,235 @@
+# Writing Your First Electron App
+
+Electron enables you to create desktop applications with pure JavaScript by
+providing a runtime with rich native (operating system) APIs. You could see it
+as a variant of the Node.js runtime that is focused on desktop applications
+instead of web servers.
+
+This doesn't mean Electron is a JavaScript binding to graphical user interface
+(GUI) libraries. Instead, Electron uses web pages as its GUI, so you could also
+see it as a minimal Chromium browser, controlled by JavaScript.
+
+**Note**: This example is also available as a repository you can
+[download and run immediately](#trying-this-example).
+
+As far as development is concerned, an Electron application is essentially a
+Node.js application. The starting point is a `package.json` that is identical
+to that of a Node.js module. A most basic Electron app would have the following
+folder structure:
+
+```text
+your-app/
+├── package.json
+├── main.js
+└── index.html
+```
+
+Create a new empty folder for your new Electron application. Open up your
+command line client and run `npm init` from that very folder.
+
+```sh
+npm init
+```
+
+npm will guide you through creating a basic `package.json` file. The script
+specified by the `main` field is the startup script of your app, which will
+run the main process. An example of your `package.json` might look like this:
+
+```json
+{
+  "name"    : "your-app",
+  "version" : "0.1.0",
+  "main"    : "main.js"
+}
+```
+
+__Note__: If the `main` field is not present in `package.json`, Electron will
+attempt to load an `index.js` (just like Node.js itself). If this was actually
+a simple Node application, you would add a `start` script that instructs `node`
+to execute the current package:
+
+```json
```

Comment:
I don't feel strongly about it, but this indented JSON style is not consistent what npm does. Any changes like `--save` would blow this formatting away.

Reply:
I'm with you. This was in original docs but is easy enough that I fixed it here 👍